### PR TITLE
As a user I should be able to view my profile

### PR DESCRIPTION
**Pull request by nkpremices**
#### What does this PR do?
Add the profile display on `/profile`

#### Description of Task to be completed?
* Design the view
* Implement the APIs logic
* Write tests

#### How should this be manually tested?
After cloning the repo: 
* Login
* Click on the avatar your
* Click on Profile

#### Screenshots (if appropriate)
Screen Shot 2019-10-16 at 20 02 25

### CODE REVIEW

**Comment by oesukam** [comment]
@nkpremices How can I access the profile page as a user?

**Comment by nkpremices** [comment]
> @nkpremices How can I access the profile page as a user?

It is written on the `How should this be manually tested` section in the PR description @oesukam

**Comment by oesukam** [comment]
@nkpremices I tried but couldn't access the page. Try to use the deployed previous link `https://deploy-preview-20--tech-community-frontend.netlify.com/`

**Comment by nkpremices** [comment]
If you're trying to login before you test it you won't see the changes cause the social login redirects you directly to the staging. Unless you put a valid token and a username directly on the local storage on `https://deploy-preview-20--tech-community-frontend.netlify.com` @oesukam

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -113,14 +117,21 @@ export class Header extends Component {
...
+              LOGIN
+            </button>
           </li>
         </ul>
       </div>
     );
   };
 
   render() {
+    // console.log(this.props);
```
Comment (oesukam) [inline src/components/Header/Header.jsx]: Can you remove this console?

File: src/components/Profile/Profile.jsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+              <p>{details && details.city ? details.city : 'City'}</p>
+            </div>
+            <div className="col-lg-4 profile__location--province">
+              <span>Province</span>
+              <p>{details && details.province ? details.province : 'Province'}</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className="col-lg-3" />
```
Comment (oesukam) [inline src/components/Profile/Profile.jsx:101]: What is this for?

File: src/components/Profile/Profile.jsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  match: {},
+};
+
+export const mapStateToProps = ({ currentUser: { details } }) => ({
+  details,
+});
+
+export default connect(
+  mapStateToProps,
+  {},
```
Comment (oesukam) [inline src/components/Profile/Profile.jsx]: You don't need to provide the same argument if you don't actions to provide

**Review by oesukam — CHANGES_REQUESTED** [review]

File: src/components/Profile/Profile.jsx (hunk 2 above)
Comment (nkpremices) [reply src/components/Profile/Profile.jsx:101]: For the right part of the profile page 
Someone else is working on it 
He will have to add his component there

**Comment by oesukam** [comment]
@nkpremices Any update?

**Comment by nkpremices** [comment]
> @nkpremices Any update?

@oesukam i've already implemented your feedback

**Review by oesukam — APPROVED** [review]

**Pull request merged**